analytics/tests/test_counts.py

```diff
@@ -1327,6 +1327,15 @@ def test_increment(self) -> None:
         do_increment_logging_stat(self.default_realm, stat, None, self.TIME_ZERO)
         self.assertTableState(RealmCount, ["value"], [[3]])
 
+    def test_do_increment_logging_start_query_count(self) -> None:
+        stat = LoggingCountStat("test", RealmCount, CountStat.DAY)
+        self.assertNumQueries(
+            4,
```

Comment:
1 select query, 1 update query, 1 save point query and 1 release that save point.
```sql
Captured queries were:
1. SELECT "analytics_realmcount"."id", "analytics_realmcount"."property", "analytics_realmcount"."subgroup", "analytics_realmcount"."end_time", "analytics_realmcount"."value", "analytics_realmcount"."realm_id" FROM "analytics_realmcount" WHERE ("analytics_realmcount"."end_time" = '1988-03-14T00:00:00+00:00'::timestamptz AND "analytics_realmcount"."property" = 'test' AND "analytics_realmcount"."realm_id" = 5 AND "analytics_realmcount"."subgroup" IS NULL) LIMIT 21
2. SAVEPOINT "s137417170173952_x7"
3. INSERT INTO "analytics_realmcount" ("property", "subgroup", "end_time", "value", "realm_id") VALUES ('test', NULL, '1988-03-14T00:00:00+00:00'::timestamptz,  -1, 5) RETURNING "analytics_realmcount"."id"
4. RELEASE SAVEPOINT "s137417170173952_x7"
```